pkg/aws/client_test.go

```diff
@@ -528,4 +533,65 @@ var _ = Describe("Client", func() {
 			),
 		)
 	})
+
+	Describe("oidc-provider", func() {
+		Context("Filtering test", func() {
+			BeforeEach(func() {
+				mockedOidcProviderArns = []string{
+					"arn:aws:iam::765374464689:oidc-provider/oidc.test1/123123123123",
+					"arn:aws:iam::765374464689:oidc-provider/oidc.test2/234234234234",
+					"arn:aws:iam::765374464689:oidc-provider/oidc.test3/345345345345",
+					"arn:aws:iam::765374464689:oidc-provider/oidc.test123/456456456456",
+				}
+				config1, err := MockOidcConfig("config1", "http://oidc.test1/123123123123")
+				Expect(err).ToNot(HaveOccurred())
+				config2, err := MockOidcConfig("config2", "http://oidc.test2/234234234234")
+				Expect(err).ToNot(HaveOccurred())
+				config3, err := MockOidcConfig("config1", "http://oidc.test3/345345345345")
+				Expect(err).ToNot(HaveOccurred())
+				mockedOidcConfigs = []*cmv1.OidcConfig{config1, config2, config3}
+				mockIamAPI.EXPECT().ListOpenIDConnectProviders(gomock.Any()).Return(&iam.
+					ListOpenIDConnectProvidersOutput{OpenIDConnectProviderList: []*iam.OpenIDConnectProviderListEntry{
+					{Arn: aws.String(mockedOidcProviderArns[0])},
```

Comment:
@hunterkepley I think we can use `awsSdk.String`  which will remove the dependency on the AWS SDK v2 which you're currently importing as `aws.